specs/TransciptTestRunner.md

```diff
@@ -37,6 +40,59 @@ Here is a high level class diagram to get started
 - `_XYZ_Client`: specialized client that knows how to interact with a specific channel (it should be capable of generating channel OAuthTokens, handle responses coming from that specific channel, etc.)
 - `ClientType` (enum): types are: DirectLine, Emulator, Teams
 
+## TestRunner Class
+### Methods
+- `RunTestAsync(string)`  Executes a single scenario based on the transcript passed to the method.   
+    * _Parameters_   
+        * transcript `string`    The transcript that contains the scenario to execute.  
+    * _Returns_  
+A `Task`.    
+
+    * Sample Usage   
+<img src="media/runtestasync1.png" width="700" border=1px>
```

Comment:
Consider using code snippets rather than bitmaps so we can edit them
```csharp
var runner = new TestRunner(new TestClientFactory(ClientType.DirectLine).GetTestClient());
await runner.RunTestAsync(Path.Combine(_transcriptsFolder, transcript));
```